config.json

```diff
@@ -11,6 +11,8 @@
 		"EMBED_SIZE": 256,
 		"LEARNING_RATE": 1e-3,
 		"IMAGE_SIZE": 224,
-		"LOG_INTERVAL": 2
+		"LOG_INTERVAL": 2,
+		"IMAGE_NET_MEANS": [0.485, 0.456, 0.406],
```

Comment:
awesome! thanks for putting this here

Reply:
Si, lo único que veo que el nombre quería que fuera IMAGENET_MEANS en lugar de IMAGE_NET_MEANS, sin el guión bajo. Pero bueno...

Reply:
Thank you!